vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -218,12 +218,6 @@ def decode_metadata(self) -> Optional["ROCmFlashAttentionMetadata"]:
             max_encoder_seq_len=self.max_encoder_seq_len,
             cross_slot_mapping=self.cross_slot_mapping,
             cross_block_tables=self.cross_block_tables)
-        # Batch may be composed of prefill|decodes, adjust query start indices
```

Comment:
Why is this section being removed?

Reply:
sorry... this is accident... it has been restored.